overlord/snapstate/snapstate.go

```diff
@@ -53,12 +53,9 @@ const (
 	// support
 	firstInterimUsableFlagValue
 
-	// RollbackMode means the given snap got rolled back
-	RollbackMode
-
-	// if we need flags for just SnapSetup it may be easier
-	// to start a new sequence from the other end with:
-	// 0x40000000 >> iota
+	// Flags for SnapSetup:
+	// RollbackOp means the given snap got rolled back
+	RollbackOp = 0x40000000 >> iota
```

Comment:
could you double check, this is correct? if I read the lang ref correctly you need a new const block to get the right effect